reference-architecture/aws-ansible/playbooks/openshift-install.yaml

```diff
@@ -34,6 +41,14 @@
       - vars/main.yaml
   roles:
       - openshift-volume-quota
+- hosts: masters
+  gather_facts: no
```

Comment:
Is gather_facts causing a big issue? If so, it might be better to leverage fact caching, at least with the python wrapper (where you can wipe the cache between invocations to ensure no bad data being pulled in between runs).